tests/python-gpu/conftest.py

```diff
@@ -58,5 +58,6 @@ def pytest_collection_modifyitems(config, items):
     # mark dask tests as `mgpu`.
     mgpu_mark = pytest.mark.mgpu
     for item in items:
-        if item.nodeid.startswith("python-gpu/test_gpu_with_dask.py"):
+        if item.nodeid.startswith("python-gpu/test_gpu_with_dask.py") or \
+                item.nodeid.startswith("python-gpu/test_spark_with_gpu/test_spark_with_gpu.py"):
```

Comment:
@hcho3 Is this setting for GPU test correct ?
What I need is:
run this file `python-gpu/test_spark_with_gpu/test_spark_with_gpu.py` in a machine with 4 GPUs.

Reply:
@WeichenXu123 Let me take a look tomorrow.

Reply:
@WeichenXu123 Yes, this setting is correct.